CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.0.1] - 2019-02-25
+
+### Features
+- Implement a new`typed-enum` rule to detect enum value that do not respect specified type [#913](https://github.com/stoplightio/spectral/pull/913)
+
+### Changed
+- API: introduced document [#876](https://github.com/stoplightio/spectral/pull/876) - note, this is **not** a breaking change
+- Introduce optional enhancers to casing function [#884](https://github.com/stoplightio/spectral/pull/884)
+
+### Fixes
+- Improved Example Object validation [#882](https://github.com/stoplightio/spectral/pull/882)
+- `oas3-operation-security-defined` rule supports optional authentication [#895](https://github.com/stoplightio/spectral/pull/895)
+- Generate more correct paths to error [#900](https://github.com/stoplightio/spectral/pull/900)
+- `example-value-or-externalValue` no longer reports false positives [#899](https://github.com/stoplightio/spectral/pull/899)
+- `schema-path` function does not bail out falsy values and JSON path expression provided as a field of rule [#917](https://github.com/stoplightio/spectral/pull/917) 
```

Comment:
@P0lip Although I know a bit about Spectral, I haven't been able to understand this part `does not bail out falsy values and JSON path expression provided as a field of rule`.

How about splitting this into two simpler changes as `Teach the schema-path function to accept a Json Path expression as a field selector` and `Make schema-path err explicitly with invalid corner case inputs`?

Reply:
Yeah, I didn't really know how to express it in a reasonable way. I wouldn't like to split it into two changes, because it's a single entry.
Any other ideas?

Reply:
@P0lip if you did two things in one PR thats ok to mention it as two changelog items. You could do 100 things in one PR and make 100 changelog items. It doesn't matter.